metadata-protobuf/proto/Membership.proto

```diff
@@ -21,6 +21,8 @@ message MembershipMetadata {
       IRC = 8;
       WECHAT = 9;
       WHATSAPP = 10;
+      VALIDATOR = 11;
```

Comment:
I don't think the validator account should be passed as an external resource. It's not really an external resource and also with the current mapping it would still appear in the QN and Pioneer as an external resource which would be confusing.

Instead I think it could just be added like this:
```proto
message MembershipMetadata {
  optional string validatorAccount = 6; // Address of the member's validator account
}
```